profiles/kentik_snmp/a10_networks/a10-thunder.yml

```diff
@@ -493,3 +515,6 @@ metric_tags:
   - column:
       name: axSysAFleXEngineVersion
       OID: 1.3.6.1.4.1.22610.2.4.1.6.4.0
+  - column:
+      name: axHAGroupID
+      OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2.1.1
```

Comment:
@al3kh - this needs to be targeted to the table; this `metric_tags` section is the global one that applies to the entire profile. There are examples of the correct indentation level and syntax throughout this profile; the first is on lines 179-183.

at a high level:

```yaml
metrics:
  - MIB: $MIB_NAME
    table: ... # table overview
    symbols: ... # metrics to collect from this table
    metric_tags: ... # attributes to decorate on top of the symbols defined in this table
...

# global metric_tags that apply to all scalar OIDs in this profile
metric_tags: ...
```